portfolio/src/main/webapp/style.css

```diff
@@ -258,6 +254,27 @@ li {
   vertical-align: top;
 }
 
-#full-comment{
-  margin-top:.4%;
+#full-comment {
+  margin-top: .4%;
+}
+
+.btn {
```

Comment:
Try to avoid abbreviations like this unless they are very widely understood.

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Style

Reply:
I'm not sure I get from the documentation why atr is not ok and nav is? What exactly is the distinction here? Bootstrap also refers to its buttons this way so I used a similar convention.

Reply:
It's definitely subjective, with some grey areas. Personally, "nav" only suggests "navigation" to me, while I would never have guessed "author" from "atr" (more likely "attribute," if anything). "btn" was unfamiliar enough for me to call it out, even though I can tell the meaning from context.